ginlong_solis_api_connector.py

```diff
@@ -217,6 +216,83 @@ def get_average_value(inverter_data, field_phase_1, field_phase_2, field_phase_3
             average_value = float((inverter_data[field_phase_1] + inverter_data[field_phase_2] + inverter_data[field_phase_3]) / 3)  # pylint: disable=line-too-long
         return average_value
 
+    def make_dict_detail_float(dict_to_change):
```

Comment:
I'd maybe add a second parameter to the method given a link of parameters to convert and iterate over this list of parameters to convert those... Something like:

```
def convert_dict_details_to_float(dict_to_change, parameters):
  for param in parameters:
    dict_to_change[param} = float(dict_to_change[param])

  return dict_to_change
```

and call it like

```
result = convert_dict_details_to_float(my_inverter_dict, ["iPv1", "iPv2", ..., "uAc3"])
```